grpclb/grpclb_test.go

```diff
@@ -771,50 +828,25 @@ func TestGRPCLBStatsUnarySuccess(t *testing.T) {
 		}
 	})
 
-	if err := checkStats(&stats, &lbmpb.ClientStats{
-		NumCallsStarted:               int64(countRPC),
-		NumCallsFinished:              int64(countRPC),
-		NumCallsFinishedKnownReceived: int64(countRPC),
+	if err := checkStats(stats, &rpcStats{
+		numCallsStarted:               int64(countRPC),
+		numCallsFinished:              int64(countRPC),
+		numCallsFinishedKnownReceived: int64(countRPC),
 	}); err != nil {
 		t.Fatal(err)
 	}
 }
 
-func TestGRPCLBStatsUnaryDropLoadBalancing(t *testing.T) {
+func TestGRPCLBStatsUnaryDrop(t *testing.T) {
 	defer leakcheck.Check(t)
 	c := 0
-	stats := runAndGetStats(t, true, false, func(cc *grpc.ClientConn) {
+	stats := runAndGetStats(t, true, func(cc *grpc.ClientConn) {
 		testC := testpb.NewTestServiceClient(cc)
 		for {
 			c++
-			if _, err := testC.EmptyCall(context.Background(), &testpb.Empty{}); err != nil {
-				if strings.Contains(err.Error(), dropErrDesc) {
-					break
-				}
+			if c%100 == 0 {
+				fmt.Println(c)
```

Comment:
Why print the value of c here, did you forget to remove it?

Reply:
Yes..
Removed.